fdk_rdf_parser_service/service/parser_service.py

```diff
@@ -46,7 +47,9 @@ async def fetch_and_parse_catalog(
 ) -> ParsedCatalog:
     """Fetch and parse catalogs."""
     async with aiohttp.ClientSession() as session:
-        rdfData = await fetch_catalog(session, catalog.uri)
+        rdfData = await fetch_catalog(
+            session, f"{FDK_REASONING_SERVICE_URI}/{catalogType}/{catalog.fdkId}"
```

Comment:
for reasoning så blir det `/datasets/catalogs/{id}`, dvs:
```suggestion
            session, f"{FDK_REASONING_SERVICE_URI}/{catalogType}/catalogs/{catalog.fdkId}"
```

Reply:
bra catch!